Content.Client/UserInterface/Systems/Chat/Widgets/ChatBox.xaml.cs

```diff
@@ -184,5 +268,22 @@ protected override void Dispose(bool disposing)
         ChatInput.Input.OnKeyBindDown -= OnInputKeyBindDown;
         ChatInput.Input.OnTextChanged -= OnTextChanged;
         ChatInput.ChannelSelector.OnChannelSelect -= OnChannelSelect;
+        _cfg.UnsubValueChanged(EECVars.ChatStackLastLines, UpdateChatStack); // EE - Chat stacking
+    }
+
+    // EE - Chat stacking
+    private sealed class ChatStackData
+    {
+        public string WrappedMessage;
+        public Color ColorOverride;
+        public int RepeatCount = 0;
+        public bool IgnoresChatstack;
```

Comment:
You can remove the entire field now. We can add it back if we ever port whatever uses it in EE